templates/gitlab-multi-runner/0/rancher-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '2'
+
+catalog:
+  name: "gitlab-multi-runner"
+  version: "1.0.0"
```

Comment:
Probably, version could be same as gitlab docker image version... 9.3.0 or used version...